docker/nginx/nginx.conf

```diff
@@ -33,6 +33,9 @@ server {
 
     add_header Access-Control-Allow-Origin "*" always;
     add_header Access-Control-Allow-Credentials "false" always;
+    add_header X-Frame-Options "SAMEORIGIN";
+    add_header X-Content-Type-Options "nosniff";
+    add_header X-XSS-Protection "1; mode=block";
```

Comment:
according to https://securityheaders.cz/x-xss-protection and https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection chrome, firefox, and edge don't use the XSS auditor, so I believe this header should not be implemented in favor of recommended CSP.
I don't insist on implementing CSP in this PR as it's not clear and easy task

Reply:
understand, I removed X-XSS-Protection. https://github.com/shopsys/shopsys/pull/3050/commits/6825ad1d988c04e27dfe9c2898a3d56c68c58322